play/src/front/Phaser/Game/MapEditor/Tools/AreaEditorTool.ts

```diff
@@ -448,6 +449,7 @@ export class AreaEditorTool extends MapEditorTool {
 
     public handleAreaCreation(config: AreaData, localCommand: boolean): void {
         this.scene.getGameMapFrontWrapper().listenAreaCreation(config);
+        this.scene.getGameMapFrontWrapper().areasManager.addArea(config);
```

Comment:
Same comment:

1. Is this normal that this method is not called in `TrashEditorTool.handleAreaCreation`.
2. Shouldn't we move the call to `areasManager.addArea` inside `listenAreaCreation`?